gke/README.md

```diff
@@ -54,9 +54,41 @@ gcloud auth login
 
 # Deploy esp service
 ./setup_esp.sh
+```
 
+1. Copy the `config.yaml` file into the `/deploy/gke` folder. Rename
```

Comment:
Nit: this list is rendering as 1 / 1 / 2 / 3 / 4 in markdown https://screenshot.googleplex.com/6J9CMRPyxudnzdz

Reply:
Good catch! Added indentation to fix.